src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -397,15 +399,6 @@ private void hutDecClicked(@NotNull final Button button)
      */
     private List<String> getStyles()
     {
-        try
```

Comment:
I removed this bit because it did work only in SP, this "scan" is only client side which mean, the player would have seen some schematic that the server is not aware of. The builder would not be able to build them anyway.

I will implement it properly with https://github.com/Minecolonies/minecolonies/issues/488 so it will work for SP with not restrictions and MP if allowed.